docs/building-deployer-helm.md

```diff
@@ -623,69 +610,71 @@ general
 [onboarding guide](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/set-up-environment#product-identifiers).
 See [this section](building-deployer.md#publishing-a-new-version)
 for more information about tags. Let's say your application has a
-track named `v6`.
+track named `v6.0.0`.
```

Comment:
1. The track should be `6.0`, which captures all `6.0.x` versions. Track can never be a specific release version.
1. Perhaps we should use `9.0` to be consistent with the current versions of WP.
1. Can you help drop the leading `v` here and below?

Reply:
Done!